### PR TITLE
feat: add filter on Common model

### PR DESCRIPTION
**Pull request by HuyLee15**

### CODE REVIEW

File: gormx/model/common.go [hunk 1]
```diff
@@ -7,9 +7,9 @@ import (
 )
 
 type Common struct {
-	ID         int64     `gorm:"primaryKey;autoIncrement:false"`
-	CreateTime time.Time `gorm:"index;autoCreateTime:nano;type:DATETIME(6) DEFAULT NOW(6)"`
-	UpdateTime time.Time `gorm:"index;autoUpdateTime:nano;type:DATETIME(6) DEFAULT NOW(6) ON UPDATE NOW(6)"`
+	ID         int64     `gorm:"primaryKey;autoIncrement:false" filter:"#"`
+	CreateTime time.Time `gorm:"index;autoCreateTime:nano;type:DATETIME(6) DEFAULT NOW(6)" filter:"=;!=;>=;<=;>;<;#"`
+	UpdateTime time.Time `gorm:"index;autoUpdateTime:nano;type:DATETIME(6) DEFAULT NOW(6) ON UPDATE NOW(6)" filter:"=;!=;>=;<=;>;<;#"`
```
Comment (levanlongktmt) [inline gormx/model/common.go:12]: hmm, with datetime I think we just need `>;>=;<;<=`

**Review by levanlongktmt — CHANGES_REQUESTED** [review]

File: gormx/model/common.go (hunk 1 above)
Comment (HuyLee15) [reply gormx/model/common.go:12]: don't we support `=`?

Comment (levanlongktmt) [reply gormx/model/common.go:12]: a chưa nghĩ ra  case nào mình cần `=`

Comment (HuyLee15) [reply gormx/model/common.go]: yes, i updated

**Review by levanlongktmt — APPROVED** [review]

**Pull request merged**